.github/workflows/CI.yml

```
@@ -18,10 +18,21 @@ jobs:
           - 'nightly'
         os:
           - ubuntu-latest
-          - macOS-latest
+          - macos-13 # Intel
```

IMO it would be better to use
```suggestion
          - macos-latest # Apple silicon
```
as a more future-proof default and exclude the combination with 1.0 below.